src/api/airdropEligibility.js

```diff
@@ -1,11 +1,9 @@
-const { get } = require('request-promise')
+const getGeneric = require('../lib/dvf/get-generic')
 
 module.exports = async (dvf, ethAddress) => {
   if (ethAddress) {
-    const response = await get(
-      `${dvf.config.api}/v1/trading/r/airdropEligibility?ethAddress=${ethAddress}`
-    )
-    return response
+    const endpoint = `/v1/trading/r/airdropEligibility?ethAddress=${ethAddress}`
+    return getGeneric(dvf, endpoint)
```

Comment:
[OPTIONAL] Now that you use `getGeneric` I think you can set `ethAddress` as third argument of the function instead of directly in the string.

Reply:
Done tx